resources/yki/migrations/021-create-quarantine-table.0.1.0.up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS quarantine
+(
+    id            BIGSERIAL PRIMARY KEY,
+    language_code CHAR(3) REFERENCES language (code) NOT NULL,
+    end_date      DATE                               NOT NULL,
+    birthdate     TEXT                               NOT NULL,
+    first_name    TEXT                               NOT NULL,
+    last_name     TEXT                               NOT NULL,
+    ssn           TEXT,
+    email         TEXT,
+    phone_number  TEXT,
+    created       TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp,
+    updated       TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp
```

Comment:
Weeklyssä tuli esiin että asianumero olisi myös hyvä saada talteen. Lisään sille vielä kentän, esim. `diary_number TEXT NOT NULL`

Reply:
Lisätty nyt.